validator/sawtooth_validator/gossip/gossip_handlers.py

```diff
@@ -23,13 +23,60 @@
 from sawtooth_validator.protobuf.network_pb2 import GossipMessage
 from sawtooth_validator.protobuf.network_pb2 import GossipBlockResponse
 from sawtooth_validator.protobuf.network_pb2 import GossipBatchResponse
+from sawtooth_validator.protobuf.network_pb2 import GetPeersRequest
+from sawtooth_validator.protobuf.network_pb2 import GetPeersResponse
 from sawtooth_validator.protobuf.network_pb2 import PeerRegisterRequest
 from sawtooth_validator.protobuf.network_pb2 import PeerUnregisterRequest
 from sawtooth_validator.protobuf.network_pb2 import NetworkAcknowledgement
+from sawtooth_validator.exceptions import PeeringException
 
 LOGGER = logging.getLogger(__name__)
 
 
+class GetPeersRequestHandler(Handler):
+    def __init__(self, gossip):
+        self._gossip = gossip
+
+    def handle(self, connection_id, message_content):
+        request = GetPeersRequest()
+        request.ParseFromString(message_content)
+        LOGGER.debug("got peers request message "
+                     "from %s. sending ack", connection_id)
+
+        self._gossip.send_peers(connection_id)
```

Comment:
I think we are sending a NETWORK_ACK as a response in every other case (on the network). I'd like to maintain that pattern for now.